doc/release_notes/v3.1.0.rst

```diff
@@ -23,6 +23,7 @@ Fixes
 
 Enhancements
 ------------
+* Python 3.12 and 3.13 now supported
```

Comment:
Could also add that 3.14 alpha is preliminary supported.

Reply:
I didn't want to over-promise, as at this point we don't even support it with numpy, so very limited.  But I'll add something with that caveat.

Reply:
Yes, we won't claim official support until the 3.14 release anyway, but this may give people a nudge to try it out (and maybe find other problems).